src/Processor/AuthorProcessor.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+/*
+ * This file is part of the BibTex Parser.
+ *
+ * (c) Renan de Lima Barbosa <renandelima@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace RenanBr\BibTexParser\Processor;
+
+/**
+ * This class includes source code adapted from the Structures_BibTex package,
+ * (c) Elmar Pitschke <elmar.pitschke@gmx.de>, included here under PHP license:
+ * http://www.php.net/license/3_0.txt
+ *
+ * @author Andre Chalom <andrechalom@gmail.com>
+ * @link https://github.com/pear/Structures_BibTex
+ */
+class AuthorProcessor
+{
+    /**
+     * Entry point for this class
+     * @param string $&value The current tag value, will be modified in-place
+     * @param $tag The current tag. This function will only process "author" tags
+     */
+    public function __invoke(&$value, $tag)
+    {
+        if (strtolower($tag) == "author") {
+            $value = $this->_extractAuthors($value);
+        }
+    }
+
+    /**
+     * Extracting the authors
+     *
+     * @access private
+     * @param string $entry The entry with the authors
+     * @return array the extracted authors
+     * @author Elmar Pitschke <elmar.pitschke@gmx.de>
+     */
+    public function _extractAuthors($entry)
```

Comment:
This method should be `private`.

This project follow the PSR-2 standard, according to it "Method names SHOULD NOT be prefixed with a single underscore to indicate protected or private visibility." http://www.php-fig.org/psr/psr-2/#methods , then the `@access private` annotation seems useless as well